addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -3487,6 +3487,47 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    QUnit.test('embedded readonly one2many with handle widget', function (assert) {
+        assert.expect(4);
+
+        this.data.partner.records[0].turtles = [1, 2, 3];
+
+        var form = createView({
+            View: FormView,
+            model: 'partner',
+            data: this.data,
+            arch:'<form string="Partners">' +
+                    '<sheet>' +
+                        '<notebook>' +
+                            '<page string="P page">' +
```

Comment:
notebook is not necessary here (i like to keep the arch as simple as possible)

Reply:
sure, I got inspiration from another test ! done